external/avro/src/main/scala/org/apache/spark/sql/avro/AvroOptions.scala

```diff
@@ -68,8 +68,10 @@ class AvroOptions(
    * If the option is not set, the Hadoop's config `avro.mapred.ignore.inputs.without.extension`
    * is taken into account. If the former one is not set too, file extensions are ignored.
    */
-  @deprecated("Use the general data source option pathGlobFilter for filtering file names", "3.0")
```

Comment:
Why remove this if it's really deprecated? I get that it will remove some compiler warnings, but, that's not super important, or can be worked around as you do elsewhere by deprecating the test methods too?